README.md

```diff
@@ -22,44 +39,49 @@ Read the full Parse Server guide here: https://github.com/ParsePlatform/parse-se
 * You now have a database named "dev" that contains your Parse data
 * Install ngrok and you can test with devices
 
-### Getting Started With Heroku + mLab Development
+## Helpful Scripts
+These scripts can help you to develop your app for Parse Server:
+
+* `npm run watch` will start your Parse Server and restart if you make any changes.
+* `npm run lint` will check the linting of your cloud code, tests and `index.js`, as defined in `.eslintrc.json`.
+* `npm run lint-fix` will attempt fix the linting of your cloud code, tests and `index.js`.
+* `npm run prettier` will help improve the formatting and layout of your cloud code, tests and `index.js`, as defined in `.prettierrc`.
+* `npm run test` will run any tests that are written in `/spec`.
+* `npm run coverage` will run tests and check coverage. Output is available in the `/coverage` folder.
 
-#### With the Heroku Button
+# Remote Deployment
+
+## Heroku
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
-#### Without It
+Alternatively, to deploy manually:
 
 * Clone the repo and change directory to it
 * Log in with the [Heroku Toolbelt](https://toolbelt.heroku.com/) and create an app: `heroku create`
 * Use the [mLab addon](https://elements.heroku.com/addons/mongolab): `heroku addons:create mongolab:sandbox --app YourAppName`
```

Comment:
This should be rewritten to reference MongoDB Atlas instead of mLab, which has been acquired. That implies that the process should be checked if it still works, if there is no Heroku mlab module anymore, otherwise the instructions as a whole may be obsolete.

Reply:
Not sure your opinion on it, but I think it’s probably not ideal to have a “click to deploy” as it encourages deploying before linking / testing cloud code / index.js. 

I guess it’s probably fine for a quick way for people to jump into Parse. 

I’ll work through the instructions and check if they’re valid. I might need your help fixing them though.

Reply:
As far as I can tell:

-Deploy to AWS button doesn’t work (just creates a new server with “sample code”
-mLab references are obsolete

Reply:
Then I think either we can fix that or remove the button.

Reply:
Can we merge this for now and I'll work through testing the instructions? Or would you like that as part of this PR (would be good to get some content on the [blog](https://github.com/parse-community/blog/pull/33))

Reply:
Sure